src/pudl/transform/eia861.py

```diff
@@ -738,6 +759,58 @@ def _remove_nerc_duplicates(entity_list):
     return full_df
 
 
+def _compare_nerc_physical_w_nerc_operational(df):
+    """Show df rows where physical nerc region does not match operational region.
+
+    In the Utility Data table, there is the 'nerc_region' index column, otherwise
+    interpreted as nerc region in which the utility is physically located and the
+    'nerc_regions_of_operation' column that depicts the nerc regions the utility
+    operates in. In most cases, these two columns are the same, however there are certain
+    instances where this is not true. There are also instances where a utility operates in
+    multiple nerc regions in which case one row will match and another row will not.
+    The output of this function in a table that shows only the utilities where the physical
+    nerc region does not match the operational region ever, meaning there is no additional
+    row for the same utlity during the same year where there is a match between the cols.
+
+    Args:
+        df (pandas.DataFrame): The utility_data_nerc_eia861 table output from the utility_data()
+            function.
+    Returns:
+        pandas.DataFrame: A DataFrame with rows for utilities where NO listed operating nerc
+            region matches the "physical location" nerc region column that's a part of the index.
```

Comment:
This function isn't currently used, but it has potential to correct or at least inform users of some of EIA's errors. 
It shows a list of utility where the nerc operating regions don't match the listed nerc region index col (aka, the "physical" nerc region). According to my contact at EIA, this means that there has been a change in NERC region that EIA's old, bookmarked NERC region data hasn't accounted for. Technically speaking, it would be correct to update these. Let me know what you think we should do about it! We could also just have a logger update / warning that shows users where there might be discrepancies in nerc region reporting.

Reply:
perhaps we should incorporate this as a check that runs inside the ops transform function and then just logs the number of these mismatched utilities.

Reply:
Thats an option, but currently there are ~2000 instances of mismatched nerc regions. That number goes down by about 500 when UNK nerc regions are excluded (they could be filled in?).